app/controllers/application_controller.rb

```diff
@@ -29,16 +29,9 @@ class ApplicationController < ActionController::Base
   prepend_before_action :add_new_relic_trace_attributes
   prepend_before_action :session_expires_at
   prepend_before_action :set_locale
-  prepend_before_action :set_x_request_url
   before_action :disable_caching
   before_action :cache_issuer_in_cookie
 
-  # Workaround that helps our JS fetch polyfill. See:
-  # https://www.npmjs.com/package/whatwg-fetch#user-content-obtaining-the-response-url
-  def set_x_request_url
-    response.headers['X-Request-URL'] = request.url
-  end
```

Comment:
:bless: so happy to see this go

Reply:
Noting that this will also cover LG-6373, I'll plan to cancel the ticket in favor of the current one.